content/en/docs/components/pipelines/user-guides/core-functions/platform-specific-features.md

```diff
@@ -1,35 +1,44 @@
 +++
-title = "Author Tasks with Platform-Specific Functionality"
-weight = 10
+title = "Use Platform-Specific Features"
+description = "Learn how to use platform-specific features in Kubeflow Pipelines in your pipelines."
+weight = 105
 +++
 
+## Overview
 
-One of the benefits of KFP is cross-platform portability. The KFP SDK compiles pipeline definitions to [IR YAML][ir-yaml] which can be read and executed by different backends, including the [Kubeflow Pipelines open source backend][oss-be] and [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines/introduction).
+One of the benefits of KFP is cross-platform portability. 
+The KFP SDK compiles pipeline definitions to [IR YAML][ir-yaml] which can be read and executed by different backends, including the Kubeflow Pipelines [open source backend][oss-be] and [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines/introduction).
```

Comment:
I think the docs must be doc-neutral. May we remove `Vertex AI` ?

Reply:
Please see https://github.com/kubeflow/website/pull/3795#issuecomment-2297324361